network/datadog_checks/network/network.py

```diff
@@ -284,7 +284,7 @@ def _check_linux(self, instance):
         proc_location = self.agentConfig.get('procfs_path', '/proc').rstrip('/')
         custom_tags = instance.get('tags', [])
 
-        if Platform.is_containerized() and proc_location != "/proc":
+        if Platform.is_containerized() and proc_location not in ("/proc", "/host/proc"):
```

Comment:
Seems there is a related condition here that might need to be changed too ?   https://github.com/DataDog/integrations-core/pull/4029/files#diff-dd651adb689f92282ee490f6dd77a033R272

If so, having `("/proc", "/host/proc")` as constant would be great.

This is the original PR: https://github.com/DataDog/integrations-core/pull/994

@JulienBalestra , you might be a better reviewer for this PR. Could you have a look ? Thx

Reply:
Updated PR to include a constant for standard proc locations